src/View3d.ts

```diff
@@ -35,10 +36,21 @@ export interface View3dOptions {
   useWebGL2?: boolean;
 }
 
+const allGlobalLoadingOptions = {
+  numChunksToPrefetchAhead: 5,
+  prefetchAlongNonPlayingAxis: false,
+  throttleArrivingChannelData: true,
+};
+
 /**
  * @class
  */
 export class View3d {
+  // TODO because View3d is basically a top level entrypoint for volume-viewer,
+  // maybe it should create the VolumeLoaderContext with options passed in.
+  // (instead of having the loaderContext created externally)
```

Comment:
Q to ask first: will we ever want the option to prevent View3d from creating a webworker and force all work to happen on the main thread?

(A: probably not)

Reply:
I think it's fine if this viewer only ever loads data via a worker.  

However, I also think it's potentially extremely useful if you can _easily_ write code to load volume data that isn't necessarily tied to rendering.  Such as for scripting some intensity calculation like you did several weeks back.  I don't care whether code like that goes through a worker behind the scenes, as long as we have a nice simple and clean interface for passing in a url and some loadspec parameters and getting data back.

Reply:
That sounds like a great argument for giving View3d control of its own load context _and_ a solid endorsement of how loading is set up otherwise. Making the actual loaders used behind the scenes available to client packages is good, but requiring clients to engage with them when they just want to load an image isn't necessary when it's easy enough to abstract away.